cli/src/commands/config.rs

```diff
@@ -150,18 +153,23 @@ pub(crate) fn cmd_config_list(
         is_overridden,
     } in &values
     {
-        // Remove overridden values.
-        // TODO(#1047): Allow printing overridden values via `--include-overridden`.
+        let mut comment_out = false;
```

Comment:
It looks like this value always equals `*is_overridden`, so can we just use that on line 172 instead? I think it still reads well. Or do will there be other reasons to comment out a line in the future?

Reply:
you right. i don't know why i though i need additional variable ...